janusgraph-test/src/main/java/org/janusgraph/graphdb/JanusGraphOperationCountingTest.java

```diff
@@ -236,6 +236,8 @@ public void testSettingProperty() {
         mgmt.makePropertyKey("foo").dataType(String.class).cardinality(Cardinality.SINGLE).make();
         finishSchema();
 
+        clopen(option(SCHEMA_CONSTRAINTS), true);
```

Comment:
how is this related to label evaluation?

Reply:
This test case was changed in the f7ee75710ed7a1f22ce5ab657a9d4eaf2bd855a3 with additional validations - https://github.com/JanusGraph/janusgraph/pull/923/files#diff-19c61b5c821437fcb295867d83c73a0c - that will fail after moving vertexLabel()/edgeLabel() methods under the constraints enabled check, unless checks are enabled explicitly.

Reply:
It would be also possible to revert changes on this test class from PR #923.

Reply:
I would not revert the changes on the test class unless the test scenario is no longer valid, but I don't see why that would be the case.